templates/rancher-airgap/overlay/opt/corral/rancher/install-rancher.sh

```diff
@@ -6,32 +6,17 @@ helm repo update
 
 CORRAL_internal_rancher_host=${CORRAL_internal_rancher_host:="${CORRAL_internal_fqdn}"}
 CORRAL_rancher_version=${CORRAL_rancher_version:=$(helm search repo rancher-latest/rancher -o json | jq -r .[0].version)}
-kubernetes_version=$(kubectl version --short | awk '/Server Version:/ {print $3}')
-minor_version=$(echo "$kubernetes_version" | cut -d. -f2)
```

Comment:
Does this impact Rancher v2.6 ? I'm not sure k8s 2.4 is dropped there

Reply:
yeah it will impact 2.6 and below. But that's not a supported version of rancher anymore afaik?

Reply:
I'm good if 2.6 hasn't long term support then. 👍🏽